### PR TITLE
Feature/827/fix filter block view

### PR DESCRIPTION
**Pull request by ArturBekhDEV**
fix filters view also fix warnings in test for FilterBarMenu component.

### CODE REVIEW

File: src/pages/find-offers/FindOffers.tsx [hunk 1]
```diff
@@ -164,23 +164,19 @@ const FindOffers = () => {
...
       <Box sx={styles.filterSection}>
-        {isDesktop ? (
-          filtersComponent
-        ) : (
-          <AppDrawer onClose={closeDrawer} open={isOpen}>
-            {filtersComponent}
-          </AppDrawer>
-        )}
+        <AppDrawer
+          anchor={isDesktop ? 'left' : undefined}
```
Comment (MickhaelSh) [inline src/pages/find-offers/FindOffers.tsx:168]: use 'right' , not undefined

Comment (MickhaelSh) [reply src/pages/find-offers/FindOffers.tsx:168]: and create enum for left right

File: src/containers/find-offer/offer-container/OfferContainer.tsx [hunk 2]
```diff
@@ -54,12 +49,7 @@ const OfferContainer: FC<OfferContainerProps> = ({
       }
     ]
     return (
-      <Grid
-        item
-        key={el._id}
-        sm={isDesktop && isFiltersOpen ? 6 : 4}
-        sx={styles.gridItem}
-      >
+      <Grid item key={el._id} sm={4} sx={styles.gridItem}>
```
Comment (Yurenko) [inline src/containers/find-offer/offer-container/OfferContainer.tsx:52]: Are you sure that you need to delete here `isDesktop && isFiltersOpen ? 6 : 4` and leave only 4 items?
On the tablet and the desktop we have square cards and as I understand we will have 3 cards in the first row and 1 card in the second row.
I attach a photo for example.
image

File: src/pages/find-offers/FindOffers.tsx (hunk 1 above)
Comment (ArturBekhDEV) [reply src/pages/find-offers/FindOffers.tsx]: fixed

File: src/types/common/enums/common.enums.ts [hunk 3]
```diff
@@ -67,3 +67,7 @@ export enum StatusEnum {
   Active = 'active',
   Closed = 'closed'
 }
+export enum DrawerAnchorEnum {
```
Comment (MickhaelSh) [inline src/types/common/enums/common.enums.ts:70]: IconPositionEnum does the same, create common enum for both

Comment (Yurenko) [reply src/types/common/enums/common.enums.ts:70]: It will be PositionEnum, I told him

Comment (Yurenko) [reply src/types/common/enums/common.enums.ts:70]: In the future we can all top and bottom

**Review by MickhaelSh — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**